README.md

```diff
@@ -16,6 +16,10 @@ source code if you already have it handy), and it will figure out all of that
 file's dependencies and sub-dependencies, passing an array of them to your
 callback. Or use the `Sync` API to get the results as a return value.
 
+Result array has non-enumerable `patterns` property with an array of globs;
+each of the found globs is joined with every path progeny looks for dependencies in.
```

Comment:
Don't fully understand what's being said on this line, is it necessary?

Perhaps the prior sentence can just end with something like

> with an array of glob patterns found in source files' `import` statements